plugins/screen-diff-plugin/src/dist/static/index.js

```diff
@@ -91,7 +190,11 @@
         onDiffTypeChange: function (event) {
             settings.save('diffType', event.target.value);
             this.render();
-        }
+        },
+    });
+    allure.api.addTestResultBlock(TestResultView, { position: 'before' });
+    allure.api.addAttachmentViewer('application/vnd.allure.screen-diff', {
```

Comment:
Custom mime types should follow `application/vnd.*` naming convention:
https://en.wikipedia.org/wiki/Media_type#Vendor_tree

Reply:
yeah, thanks for mentioning that

Reply:
added support of `application/vnd.allure.screen-diff` to EE version as well, so we good to go

Reply:
oh, let's change it to `application/vnd.allure.image.diff`

Reply:
fixed